src/main/java/com/treasuredata/client/AbstractTDClientBuilder.java

```diff
@@ -62,6 +64,9 @@
     protected int connectTimeoutMillis = 15000;
     protected int idleTimeoutMillis = 60000;
     protected int connectionPoolSize = 64;
+    protected Multimap<String, String> headers = ImmutableMultimap.of();
+    protected Optional<Integer> requestBufferSize = Optional.absent();
+    protected Optional<Integer> responseBufferSize = Optional.absent();
```

Comment:
That is a good question. The reason we need this is to increase the request buffer size to accommodate passing on some td console session headers from workflow-api to td-api. And they are big enough to cause http requests to fail with a "too large request" error when using default buffer sizes.